samples/dotnet-azurefunction/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM mcr.microsoft.com/dotnet/sdk:6.0 AS installer-env
 
-# Copy the Dapr.AzureFunctions.Extension, style cop, and dotnet sample into the installer-env to build
-COPY /src/Dapr.AzureFunctions.Extension /src/src/Dapr.AzureFunctions.Extension 
+# Copy the Microsoft.Azure.WebJobs.Extension.Dapr, style cop, and dotnet sample into the installer-env to build
+COPY /src/Microsoft.Azure.WebJobs.Extension.Dapr /src/src/Microsoft.Azure.WebJobs.Extension.Dapr 
```

Comment:
Polite enquiry: I understand this was written earlier, but why do have /src/src folder? Any specific reason?

Reply:
Yes, so everything in the codebase is moved under the `src` directory as a Dockerfile convention, including our own `src` directory.
```
src
- .stylecop
- samples
   - dotnet-azurefunction
- src
   - Microsoft.Azure.WebJobs.Extension.Dapr
```